src/OpenLoco/Vehicles/VehicleManager.cpp

```diff
@@ -0,0 +1,16 @@
+#include "VehicleManager.h"
+#include "../Company.h"
+#include "../Interop/Interop.hpp"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::VehicleManager
+{
+    // 0x004C3A0C
+    void calculateAvailableVehicles(company& company)
```

Comment:
What about `determine` or `update`? 'calculate' sounds kinda off to me.

Reply:
`determineAvailableVehicles` sounds better to me, too, yeah.